core/src/main/scala/io/finch/internal/ToResponse.scala

```diff
@@ -71,6 +71,11 @@ trait HighPriorityToResponseInstances extends LowPriorityToResponseInstances {
   ): Aux[AsyncStream[A], Text.Plain] =
     asyncResponseBuilder((a, cs) => e(a, cs).concat(newLine))
 
+  implicit def sseAsyncToResponse[A](implicit
```

Comment:
I just want to make sure we separate SSE from finch-core, because otherwise it doesn't actually make a lot of sense to extract that into t a separate package. We can try, if it's now working, I'm happy to go back to an initial idea where SSE is defined as part of finch-core.